### DISCUSSION TITLE
Seems justify-center, px-0, pe-0 all stopped working on ui.card() inside ui.row()

### DISCUSSION THREAD

**Discussion by TotoWang-hhh** [zauberzeug/nicegui#2388] [Q&A] [answered]
### Question

I've tried every solution I found on the Internet, but nothing happens...

```python
with ui.column().style("width:100%;max-width:720px;align-self:center;background:#f0f0f0"):
    ui.label('TITLE').style("align-self:center;font-weight:bolder")
    with ui.row().props("justify-center").style(add="width:95%;align-self:center"):
        with ui.card().props("px-0 pe-0").style("background-color:#eeeeff;width:30%"):
            ui.label("Text 1").style("font-weight:bolder;align-self:center")
        with ui.card().props("px-0 pe-0").style("background-color:#eeeeff;width:30%"):
            ui.label("Text 2").style("font-weight:bolder;align-self:center")
        with ui.card().props("px-0 pe-0").style("background-color:#eeeeff;width:30%"):
            ui.label("Text 3").style("font-weight:bolder;align-self:center")
```

![image](https://github.com/zauberzeug/nicegui/assets/67090055/d6ee484c-9d4e-4adc-a904-af7049e5af58)

**Comment by falkoschindler** [comment, marked as answer]
Hi @TotoWang-hhh,

There's quite a lot going on in your code. Can you simplify it to the bare minimum?

I tried three cards in a row and "justify-center" is working:
```py
with ui.row().classes('w-96 justify-center bg-blue-100'):
    ui.card().classes('w-20')
    ui.card().classes('w-20')
    ui.card().classes('w-20')
```

![Screenshot 2024-01-17 at 16 21 11](https://github.com/zauberzeug/nicegui/assets/5767091/1e57e291-c0cb-4a43-878c-6c089d5057cf)

The problem might be that "justify-center" isn't a Quasar prop, but a Tailwind class. Therefore you need to set it using `.classes` instead of `.props`.

**Reply by TotoWang-hhh** [reply]
Well, it does work. So the truth is that I've mixed `classes()` and `props()`.
